src/features/profile/DangerCard.js

```diff
@@ -65,8 +66,18 @@ const DangerCard = () => {
 
     setIsPending(false)
   }
+  const disabled2 = entries.length > 0
```

Comment:
**suggestion:** Also disable when pending, you need to use and or for that `||`
**suggestion:** Rename variable `disabledMensenTags` or something like that more descriptive.